spring-rabbit/src/main/java/org/springframework/amqp/rabbit/connection/CachingConnectionFactory.java

```diff
@@ -174,6 +176,23 @@ public CachingConnectionFactory() {
 		this((String) null);
 	}
 
+	/**
+	 * Create a new CachingConnectionFactory given a host name.
+	 * @param hostname the host name to connect to
```

Comment:
`@since 1.7.6` ?

Reply:
This is an existing CTOR; I moved it to comply with our normal ordering. See the deleted code below.